pages/dkp/kommander/2.2/DKP-insight/architecture-overview/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Insight architecture overview
+title: Insight architecture overview
+excerpt: Insight architecture overview
+menuWeight: 10
+beta: true
```

Comment:
Instead of beta you should now be able to turn on the "techPreview" badge here with `techPreview: true`.